pontoon/base/models/user.py

```diff
@@ -220,6 +224,49 @@ def has_approved_translations(self):
     return self.translation_set.filter(approved=True).exists()
 
 
+@property
+def badges_translations_count(self):
+    """Contributions provided by user that count towards their badges."""
+    from pontoon.actionlog.models import ActionLog
+
+    return ActionLog.objects.filter(
+        performed_by=self,
+        action_type="translation:created",
+        created_at__gte=badges_inception,
+    ).count()
+
+
+@property
+def badges_reviewed_translations(self):
+    """Translation reviews provided by user that count towards their badges."""
+    from pontoon.actionlog.models import ActionLog
+
+    approvals = ActionLog.objects.filter(
+        performed_by=self,
+        action_type="translation:created",
+        created_at__gte=badges_inception,
+    ).count()
+    rejections = ActionLog.objects.filter(
+        performed_by=self,
+        action_type="translation:rejected",
+        created_at__gte=badges_inception,
+    ).count()
```

Comment:
We should be able to turn this into a single DB query by performing an OR query:
https://djangocentral.com/how-to-perform-or-queries-in-django-orm/

And we should also use `self.actions...`.